Configurations.md

```diff
@@ -2428,6 +2428,68 @@ Break comments to fit on the line
 // commodo consequat.
 ```
 
+## `wrap_long_multiline_chains`
+
+Wrap the right hand side of a let binding when it is a long multiline chain.
+Leave other right hand expressions alone.
+
+- **Default value**: false
+- **Possible values**: true, false
+- **Stable**: Yes
+
+### Example
+
+#### `false` (default):
+```rust
+fn main() {
+    let chain = my_struct
+        .map(|e| e + 1)
+        .and_then(|e| e + 1)
+        .and_then(|e| e + 1)
+        .and_then(|e| e + 1)
+        .and_then(|e| e + 1)
+        .collect();
+
+    let struct_let = Struct {
+        item_one,
+        item_two,
+        item_three,
+        item_four,
+        item_five,
+        item_six,
+        item_seven,
+        item_eight,
+    };
+}
+```
+
+#### `true`:
+```rust
+fn main() {
+    let chain =
+        my_struct
+        .map(|e| e + 1)
+        .and_then(|e| e + 1)
+        .and_then(|e| e + 1)
+        .and_then(|e| e + 1)
+        .and_then(|e| e + 1)
+        .collect();
```

Comment:
The [style guide for multiline chains](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/expressions.md#chains-of-fields-and-method-calls) requires that the the chain items be indented from the parent. 

The user's configured indentation style (as specified via `indent_style`) also has to be respected, but the indentation style would be broken when `wrap_long_multiline_chains` is true with both `Visual` and `Block` indent styles. 

For example today:
```rust
// indent_style = "Visual"
fn foo() {
    let chain = my_struct.map(|e| e + 1)
                         .and_then(|e| e + 1)
                         .and_then(|e| e + 1)
                         .and_then(|e| e + 1)
                         .and_then(|e| e + 1)
                         .collect();
}
```

which would be broken with `wrap_long_multiline_chains`:

```rust
// indent_style = "Visual"
// wrap_long_multiline_chains = true
fn foo() {
    let chain =
        my_struct.map(|e| e + 1)
                         .and_then(|e| e + 1)
                         .and_then(|e| e + 1)
                         .and_then(|e| e + 1)
                         .and_then(|e| e + 1)
                         .collect();
}
```